File: DevBuildFinal_LMS/Models/Course.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DevBuildFinal_LMS.Models
{
    public class Course
    {
        public int courseId { get; set; }
        public string courseName { get; set; }
        public int assignedTeacherId { get; set; }
    }

    public class StudentCourse
    {
        public int courseId { get; set; }
        public int studentId { get; set; }
    }
}

File: DevBuildFinal_LMS/ClientApp/src/app/interfaces/course.ts
export interface Course {
  courseId: number;
  courseName: string;
  assignedTeacherId: number;
}

export interface Module {
  moduleId: number;
  moduleName: string;
  courseId: number;
}

export interface Assignments {
  assignmentId: number;
  assignmentName: string;
  moduleId: number;
  assignmentURL: string;
}

export interface Resource {
  resourceId: number;
  resourceName: string;
  moduleId: number;
  resourceURL: string;
}

export interface NewCourse {
  courseName: string;
}

export interface StudentCourse {
  studentId: number;
  courseId: number;
}
